src/main/java/ch/njol/skript/effects/EffForceEnchantmentGlint.java

```diff
@@ -0,0 +1,78 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Force Enchantment Glint")
+@Description("Forces the items to glint or not.")
+@Examples({
+	"force {_items::*} to glint",
+	"force the player's tool to stop glinting"
+})
+@RequiredPlugins("Spigot 1.20.5+")
+@Since("INSERT VERSION")
+public class EffForceEnchantmentGlint extends Effect {
+
+	static {
+		Skript.registerEffect(EffForceEnchantmentGlint.class,
+			"(force|make) %itemtypes% to [start] glint[ing]",
+			"(force|make) %itemtypes% to (not|stop) glint[ing]");
```

Comment:
```suggestion
			"(force|make) %itemtypes% [to] [start] glint[ing]",
			"(force|make) %itemtypes% [to] (not|stop) glint[ing]");
```
"to" is only appropriate for "force", not "make"